docs/optimization_guide/nncf/ptq/quantization_w_accuracy_control.md

```diff
@@ -51,14 +51,33 @@ Now, you can run quantization with accuracy control. The following code snippet
       :fragment: [quantization]
 
 
-``max_drop`` defines the accuracy drop threshold. The quantization process stops when the degradation of accuracy metric on the validation dataset is less than the ``max_drop``.
+* ``max_drop`` defines the accuracy drop threshold. The quantization process stops when the degradation of accuracy metric on the validation dataset is less than the ``max_drop``. The default value is 0.01.
+
+* ``drop_type`` defines how the accuracy drop will be calculated: ``ABSOLUTE`` (used by default) or ``RELATIVE``.
+
+After that the model can be compiled and run with OpenVINO:
+
+.. tab:: OpenVINO
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_aa_openvino.py
+      :language: python
+      :fragment: [inference]
+
+
+``nncf.quantize_with_accuracy_control()`` API supports all the parameters from :doc:`Basic 8-bit quantization <basic_quantization_flow>` API and you can also use them to quantize a model with accuracy control and a custom configuration.
```

Comment:
```suggestion
``nncf.quantize_with_accuracy_control()`` API supports all the parameters from :doc:`Basic 8-bit quantization <basic_quantization_flow>` API, to quantize a model with accuracy control and a custom configuration.
```
I had to read this sentence a few times. I think it's clearer without the "and you can also use them" part. Maybe someone has a better suggestion too.

Reply:
updated as you suggested, I also think it's better